juniper_official/Solutions/Traffic-Blackhole/interface-interrups.rule

```diff
@@ -0,0 +1,112 @@
+/*
+	* REQ: COL-1109
+	* Rule to collect Interface Block CM Errors and Interrupts for Blackhole Detection RCA.
+*/
+healthbot {
+	topic hardware.fpc {
+		rule interface-interrupts {
+			keys [ fpc-name error-name ];
+			synopsis "Interface Block CM Errors and Interrupts";
+			description "Interface Block CM Errors and Interrupts";
+			sensor sensor-interrupts {
+				synopsis "Interface Block CM Errors and Interrupts";
+				description "Interface Block CM Errors and Interrupts";
+				open-config {
+					sensor-name /components/component/integrated-circuit/pipeline-counters/errors/interface-block/interface-block-error/;
```

Comment:
This needs to be changed to /components/component

Reply:
Changed path to: /components/component/